Src/IronPython/Runtime/Operations/StringOps.cs

```diff
@@ -286,7 +290,7 @@ public static class StringOps {
         }
 
         [StaticExtensionMethod]
-        public static object __new__(CodeContext/*!*/ context, PythonType cls, [BytesLike, NotNull]IList<byte> @object, [NotNull]string encoding, [NotNull, DisallowNull]string errors = null) {
+        public static object __new__(CodeContext/*!*/ context, [NotNull]PythonType cls, [BytesLike, NotNull]IList<byte> @object, [NotNull]string encoding, [NotNull, DisallowNull]string? errors = null) {
             if (cls == TypeCache.String) {
                 if (@object is Bytes) return ((Bytes)@object).decode(context, encoding, errors);
```

Comment:
I used `null` as default here because it avoids the switch on string, but I have an idea to postpone the whole switch until a decoding error occurs. Then this little efficiency will not matter, overshadowed by the exception handling costs. When done, I will probably replace `null` with `"strict"` as default everywhere in `_codecs` as it is more readable.